concepts/interfaces/about.md

```diff
@@ -1,3 +1,53 @@
-# About
+# About Interfaces
 
-TODO: add information on interfaces concept
+An **interface** is effectively a set of method signatures.
+Any value that defines those methods "implements" the interface type (implicitly).
+Here is what an interface definition might look like:
+
+```go
+type InterfaceName interface {
+    MethodOne() MethodOneReturnType
+    ...
+}
+```
+
```

Comment:
I would have liked two more pieces of info in this section.
1. that a type can implement multiple interfaces
2. what are interfaces good for/ when are they used

(inspired by https://golang.org/doc/effective_go#interfaces)

Here some possible text but feel free to write your own if you don't like it.
```suggestion

Since interfaces are implemented implicitly, a type can easily implement multiple different interfaces.

If all function/method parameters and return values would need to be of one specific concrete (ie. non-interface) type, that would strongly limit their re-usability.
By using interface types for the parameters, only the behavior that is needed for the function to do its job is defined.
Now the function can be used with parameters of different concrete types, as long as they implement the correct interface.
With this, you can for example achieve dependency injection in Go.
A similar logic applies when using interface types as return values as we will see in the example below.

```